### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by VinayGuthal**
FolderOrigin-RevId: /google/src/cloud/vguthal/update_copybara_firestore/.

### CODE REVIEW

File: firebase-firestore/src/main/java/com/google/cloud/datastore/core/number/IndexNumberDecoder.java [hunk 1]
```diff
@@ -1,17 +1,3 @@
-// Copyright 2018 Google LLC
```
Comment (wilhuff) [inline firebase-firestore/src/main/java/com/google/cloud/datastore/core/number/IndexNumberDecoder.java]: We shouldn't drop these notices.

File: firebase-firestore/src/main/java/com/google/cloud/datastore/core/number/NumberComparisonHelper.java [hunk 2]
```diff
@@ -77,25 +59,14 @@ public static int compareLongs(long leftLong, long rightLong) {
     }
   }
 
-  /** Compares ints. See the comment to {@link #compareLongs} for rationale. */
-  public static int compareInts(int leftInt, int rightInt) {
```
Comment (wilhuff) [inline firebase-firestore/src/main/java/com/google/cloud/datastore/core/number/NumberComparisonHelper.java]: This is still in use so we can't remove it (though I guess we could also migrate the caller to calling compareLongs)

Comment (VinayGuthal) [reply firebase-firestore/src/main/java/com/google/cloud/datastore/core/number/NumberComparisonHelper.java]: Yes. this was a part of a refractor. Sorry for creating the pr too early.

**Pull request merged**